src/statmodels.jl

```diff
@@ -160,17 +160,50 @@ df_residual(obj::RegressionModel) = error("df_residual is not defined for $(type
 ## Nms are the coefficient names, corresponding to rows in the table
 type CoefTable
     mat::Matrix
+    cols::Tuple
```

Comment:
In Julia `[v1, v2, ...]` will no longer concatenate, and we could have a temporary convenience method if needed. Anyway, I'm all for using a tuple, but then could you change other fields to tuples too?